static/js/amp_store.js

```diff
@@ -0,0 +1,49 @@
+window.onload = function(){
+  let optin = document.getElementById("amp_store");
+  let ampPvf = document.getElementById("amp_pvf");
+  if (optin.value == '' || optin.value == false){
+    ampPvf.removeAttribute('readonly');
+    optin.removeAttribute('onclick');
+  }
+}
+
+function handleAmpStoreProps() {
+  let inputDataClass = "";
+  let outputDataClass = "";
+  const privateData = ["L1", "L2"];
+  const publicData = ["L3", "L4"];
+  const pvfgig200 = "GIG 200";
+
+  const inputData = document.getElementById("input_data_class");
+  if (inputData != null){
+      inputDataClass = inputData.value.substr(0,2);
+  }
+
+  const outputData = document.getElementById("output_data_class");
+  if (outputData != null){
+      outputDataClass = outputData.value.substr(0,2);
+  }
+
+  if (inputDataClass != "" && outputDataClass != ""){
+    let optin = document.getElementById("amp_store");
+    let ampPvf = document.getElementById("amp_pvf");
+
+    const dataAccess = [inputDataClass, outputDataClass].sort()[0];
+    if (publicData.includes(dataAccess)){
+        // disable setting optin and amp pvf
+        // set default amp pvf to GIG 200
+        optin.checked = true;
+        optin.setAttribute('onclick', "return false;");
+        ampPvf.value = pvfgig200;
+        ampPvf.setAttribute('readonly', true);
+    } else if (privateData.includes(dataAccess)){
```

Comment:
Looks like now we do not default opt-out L1 projects anymore, is this expected?

Reply:
imo, giving same treatment to private data would help users to have consistent behavior. Good point; will confirm with team.

Reply:
updated code to have opt-out unchecked by default for L1. Thanks!